cni/network/network.go

```diff
@@ -422,37 +422,38 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 
 		// Add Interfaces to result.
 		// previously just logged the default (infra) interface so this is equivalent behavior
-		cniResult := &cniTypesCurr.Result{}
+		cniResults := []cniTypesCurr.Result{}
 		for key := range ipamAddResult.interfaceInfo {
 			logger.Info("Exiting add, interface info retrieved", zap.Any("ifInfo", ipamAddResult.interfaceInfo[key]))
 			// previously we had a default interface info to select which interface info was the one to be returned from cni add
```

Comment:
if the cniresult logic is being modified can you update this comment?

Reply:
yes, before we only stdout one cniResult and containerd creates one pod, now we should provide multiple cniResults for containerd to create multiple pods